spec/unit/trackable_spec.rb

```diff
@@ -298,6 +298,22 @@ def my_changes
         expect(m).to receive(:my_changes).once.and_call_original
         m.save
       end
+
+      it 'should allow an alternate method to be specified on object creation' do
+        class CustomTracker < MyModel
```

Comment:
note that this class isn't actually scoped to the `it`, if you really want to make it better, declare it via

```
let(:custom_tracker) { Class.new(MyModel) do ... }
```

I know we don't do it elsewhere, so I am not nitpicking on it.